src/main/resources/templates/ShoppingCart.html

```diff
@@ -63,8 +63,12 @@ <h1>Shopping cart</h1>
             Pizza
             <span class="badge badge-primary badge-pill">50</span>
             <div>tomato or alfredo sauces.
+                <!-- As a recommendation, you can have the 'add' button trigger a Modal to select the specifics for each type of food.
+                This helps prevent the unusual text display as seen here. This however is a design choice, and the course is more about
+                backend than front end development.-->
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.